addon/styles/addon.scss

```diff
@@ -46,5 +54,11 @@
   .animate {
     transition: transform 0.2s ease;
   }
+}
 
+.frost-sort-container {
```

Comment:
If I am reading this right, `frost-sort-container` is a div inside frost-sort so this CSS should be within the `frost-sort ` block, not outside.